app/controllers/restaurant_results_controller.rb

```diff
@@ -37,50 +76,50 @@ def update
   private
 
   def set_result
-    cuisine_candidates = set_cuisine_candidates
-    # year_candidates = set_year_candidates
-    # mood_candidates = set_mood_candidates
-    # weather_candidates = set_weather_candidates
-    star_candidates = set_star_candidates
-
-    # (cuisine_candidates + mood_candidates + year_candidates + weather_candidates + star_candidates).sample
-    (cuisine_candidates + star_candidates).sample
+    # cuisine_candidates = set_cuisine_candidates
+    # # year_candidates = set_year_candidates
+    # # mood_candidates = set_mood_candidates
+    # # weather_candidates = set_weather_candidates
+    # star_candidates = set_star_candidates
+
+    # # (cuisine_candidates + mood_candidates + year_candidates + weather_candidates + star_candidates).sample
+    # (cuisine_candidates + star_candidates).sample
   end
 
   def set_star_candidates
-    star_sign = params[:star] if params[:star]
-    star_sign ? star_sign(star_sign) : []
+    # star_sign = params[:star] if params[:star]
+    # star_sign ? star_sign(star_sign) : []
   end
 
   def star_sign(star_sign)
-    first_letter = star_sign[0]
-    Restaurant.where("title ilike '#{first_letter}%'")
+    # first_letter = star_sign[0]
+    # Restaurant.where("title ilike '#{first_letter}%'")
   end
 
   def set_restaurant
-    @restaurant_result = RestaurantResult.where(user: current_user).last
+    # @restaurant_result = RestaurantResult.where(user: current_user).last
   end
 
   def set_options
-    @restaurant_cuisines = []
-    @years_for_select = [["60s", 1960], ["70s", 1970], ["80s", 1980], ["90s", 1990], ["2000s", 2000], ["2010s", 2010], ["2020s", 2020]]
+    # @restaurant_cuisines = []
+    # @years_for_select = [["60s", 1960], ["70s", 1970], ["80s", 1980], ["90s", 1990], ["2000s", 2000], ["2010s", 2010], ["2020s", 2020]]
   end
 
   def set_cuisine_candidates
-    cuisine = @colour_cuisine_key[params[:colour].to_sym] if params[:colour]
-    cuisine ? Restaurant.where(cuisine: cuisine) : []
+    # cuisine = @colour_cuisine_key[params[:colour].to_sym] if params[:colour]
+    # cuisine ? Restaurant.where(cuisine: cuisine) : []
   end
 
   def colour
     @colour_cuisine_key = {
-      orange: "Chinese", brown: "Mexican", pink: "Jamaican", blue: "Drama", green: "Italian",
-      red: "Taiwanese", yellow: "Spanish", purple: "French", violet: "German", emerald: "Eritrean",
-      grey: "Russian", jasmine: "Japanese", myrtle: "Greek", navy: "Vietnamese", maroon: "Polish", cream: "American"
+      orange: "French", brown: "Chinese", pink: "Chinese", blue: "Italian", green: "Greek",
```

Comment:
Chinese is in twice, I can update this no worries, unless this was done purposefully?